src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -181,7 +207,7 @@ private static void StopBridge(CommandLineArguments commandLineArgs)
                 Console.WriteLine("A problem was encountered stopping the Bridge:{0}{1}",
                                     Environment.NewLine, problem);
                 Console.WriteLine("Forcing local resource cleanup...");
-                BridgeController.StopBridgeProcess(1);
+                BridgeController.StopBridgeProcess(Bridge_ExitCode_Stopped);
```

Comment:
Reverted to only 0 (success), -1 (failure) and 1 (-require found Bridge already running).